### ISSUE TITLE
Swift 3.0 support?

### ISSUE THREAD

**Issue by shmakovigor** [ipraba/EPContactsPicker#27]

**Comment by Sorix** [comment]
I’ve added Swift 2.3 support in pull request #26

**Comment by Dark0rion** [comment]
update to swift 3 please!!

**Comment by Sorix** [comment]
Please check if that works for you. I haven't got too much time to test, so haven't opened pull request here.

```ruby
pod 'EPContactsPicker', :git => 'https://github.com/Sorix/EPContactsPicker', :branch => 'swift3'
```

**Comment by Dark0rion** [comment]
@Sorix thanks for taking the time to review the code and make a swift 3, will try to test the code later, again thank you very much!

**Comment by SkylerW** [comment]
You can use this https://github.com/SkylerW/EPContactsPicker fork for swift 3

**Comment by Sorix** [comment]
@SkylerW problem that your fork has critical issues from PR #26

**Comment by SkylerW** [comment]
Yeah, was just trying to get the ball rolling ;)

**Comment by Sorix** [comment]
Finally updated Travis, updated README, fixed crash bugs in PR #30, I will be happy for any feedback (even everything is working, I don't use Swift 3 in production projects yet myself).
